oap-server/server-starter/src/main/resources/otel-rules/elasticsearch/elasticsearch-cluster.yaml

```diff
@@ -34,39 +34,39 @@ metricPrefix: meter_elasticsearch_cluster
 metricsRules:
   # cluster health
   - name: health_status
-    exp: elasticsearch_cluster_health_status.valueEqual(1).sum(['cluster' , 'color'])
+    exp: elasticsearch_cluster_health_status.tagNotEqual('cluster','unknown_cluster').valueEqual(1).sum(['cluster' , 'color'])
```

Comment:
Attaching the source codes of the exporter, just FYI to others

![image](https://github.com/apache/skywalking/assets/5441976/b71eb157-ff78-42d9-860b-764a933a9fed)

Reply:
This `shouldn't` thing keeps happening in our demo env. We have reasons to believe this is still a case.
Adding this to remove illegal data.